code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -34,8 +34,11 @@
 			src << "<span class='warning'>Your powers are useless on this holy ground.</span>"
 			return 0
 	if(check_holy(src) && !fullpower)
-		var/turf/simulated/T = get_turf(src)
-		if(T.lighting_lumcount > 2)
+		var/atom/movable/lighting_overlay/L
+		for(var/atom/movable/lighting_overlay/LF in get_turf(src))
```

Comment:
No I totally agree with it. I just removed this proc http://puu.sh/hPjF6/bf38ebce3a.png, because it cost more than rebuilding the powernet twice just to find an APC in contents.